core/proto/keytransparency_v1_types/keytransparency_v1_types.proto

```diff
@@ -236,3 +236,19 @@ message GetMutationsResponse {
   // results.
   string next_page_token = 7;
 }
+
+// GetMonitoringRequest contains the input parameters of the GetMonitoring APIs.
+message GetMonitoringRequest {
+  // start specifies the start epoch number from which monitoring results will
+  // be returned (ranging from [start, latestObserved] and starting at 1).
+  int64 start = 1;
+}
+
+message GetMonitoringResponse {
+  // smr contains the map root for the sparse Merkle Tree signed with the
```

Comment:
I've added proto messages for the above cases here: https://github.com/google/keytransparency/pull/709/commits/5ed5f8a63dcdef8b374c79ea6ce4cc29e0304576

Let me know what you think.